CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.1.0dev - [05/03/2022]
+## v1.1.0dev - [12/12/2022]
+
+- add support for MseI.
+- add TAD, AB compartments, APA analysis
```

Comment:
Seems a bit thin for such a big PR

Reply:
Yes. I did development for about half year. :)

Reply:
Figured as much given the size of the PR.
Would have been easier to review with more tiny PRs.
We usually do big PRs like that only for releases.

Reply:
Yes, it is my fault. But I don't know how to roll it back. Will it be a big concern?

Reply:
No, don't worry, I'll finish the review.
Try just not to do that again